lib/utils/getEnsureArtifact.js

```diff
@@ -1,39 +1,16 @@
 'use strict';
```

Comment:
Let's rename the module to `ensureArtifact.js` (it was `getEnsureArtifact` as invocation was returning `ensureArtifact` function, now `ensureArtifact` will be exported directly)